src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -230,25 +277,42 @@ void testIoUpdateMedicalInventory() throws Exception {
 	}
 
 	@Test
-	void testMgrDeleteMedicalInventory() throws Exception {
+	void testDeleteMedicalInventoryWithInventoryRowsWithoutNewLot() throws Exception {
 		Integer id = setupTestMedicalInventory(false);
 		MedicalInventory foundMedicalInventory = medIvnIoOperationRepository.findById(id).orElse(null);
 		assertThat(foundMedicalInventory).isNotNull();
-		String reference = foundMedicalInventory.getInventoryReference();
-		medicalInventoryManager.deleteMedicalInventory(foundMedicalInventory);
-		assertThat(medicalInventoryManager.referenceExists(reference)).isFalse();
+		MedicalType medicalType = testMedicalType.setup(false);
+		Medical medical = testMedical.setup(medicalType, false);
+		Lot lot = testLot.setup(medical, false);
+		medicalType = medicalTypeIoOperationRepository.save(medicalType);
```

Comment:
`medicalType` is not used in this method after this assignment; can be removed

Reply:
Why did you mark this resolved?   What is the purpose of creating an object that is never used?  It doesn't add to the readability of the method.